src/index.html

```diff
@@ -2,11 +2,265 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops&computers" class="nav__link"
+            data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link ">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link ">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link ">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link ">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-content">
```

Comment:
Add blank line between the blocks for better readability
```suggestion
    </header>
    
    <main class="main-content">
```